vendor/brave-ios/Ledger/Data/BATLedgerDatabase.mm

```diff
@@ -128,10 +141,38 @@ + (void)insertOrUpdatePublisherInfo:(BATPublisherInfo *)info context:(nullable N
     pi.url = [NSURL URLWithString:info.url];
     pi.provider = info.provider;
     pi.faviconURL = [NSURL URLWithString:info.faviconUrl];
-  }];
+  } completion:WriteToDataControllerCompletion(completion)];
 }
 
-+ (void)restoreExcludedPublishers
++ (NSArray<BATPublisherInfo *> *)excludedPublishers
+{
+  const auto context = DataController.viewContext;
+  const auto fetchRequest = PublisherInfo.fetchRequest;
+  fetchRequest.entity = [NSEntityDescription entityForName:NSStringFromClass(PublisherInfo.class)
+                                    inManagedObjectContext:context];
+  fetchRequest.predicate = [NSPredicate predicateWithFormat:@"excluded = 1"];
```

Comment:
Can we use `BATExcludeFilterFilterExcluded` here? to replace that magic number with a constant

Reply:
Yeah we could, I just the value from brave-core :P